src/Microsoft.ML.Legacy/Runtime/EntryPoints/JsonUtils/ExecuteGraphCommand.cs

```diff
@@ -16,11 +16,14 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
+#pragma warning disable 612
 [assembly: LoadableClass(typeof(ExecuteGraphCommand), typeof(ExecuteGraphCommand.Arguments), typeof(SignatureCommand),
     "", "ExecGraph")]
+#pragma warning restore 612
 
 namespace Microsoft.ML.Runtime.EntryPoints.JsonUtils
 {
+    [Obsolete]
     internal sealed class ExecuteGraphCommand : ICommand
```

Comment:
>ExecuteGraphCommand [](start = 26, length = 19)

*General* entry-point utilities like this that were incorrectly moved into this doomed project must be borne out. This includes, at least, this command, the graph runner, and the manifest utils.

So let's draw a distinction, there's the CSharpApi that was generated out of entry-points. *That's* bad and needs to get removed. The facilities though for entry-points by themselves are *not* bad and should be preserved. So any "viral" effects from any references to the contents of `CSharpApi.cs` or any other similarly structured classes (e.g., the duplicate evaluators that were written meant to be used within these pipelines) should be marked obsolete. Similar story for the code-gen. But this by itself should *not* have been marked obsolete.

Reply:
Ok. I will only mark things referring to CSharpApi.cs obsolete and move entry point utilities to a new project, Microsoft.ML.EntryPoint.

---
In reply to: [235447577](https://github.com/dotnet/machinelearning/pull/1686#discussion_r235447577) [](ancestors = 235447577)